src/test/java/com/crowdin/client/clients/ClientsApiTest.java

```diff
@@ -33,4 +38,49 @@ public void listClientsTest() {
         assertEquals(clientResponseList.getData().get(0).getData().getName(), name);
         assertEquals(clientResponseList.getData().get(0).getData().getStatus(), Status.PENDING);
     }
+
```

Comment:
Also wrong location of the testing methods. Please remove the unnecessary comments and follow the rest of the coding style.